packages/generator/ni_measurement_plugin_sdk_generator/client/__init__.py

```diff
@@ -47,8 +47,215 @@ def _create_file(
         file.write(formatted_output)
 
 
+def _resolve_output_directory(directory_out: Optional[str]) -> pathlib.Path:
+    if directory_out is None:
+        directory_out_path = pathlib.Path.cwd()
+    else:
+        directory_out_path = pathlib.Path(directory_out)
+
+    if not directory_out_path.exists():
+        raise click.ClickException(f"The specified directory '{directory_out}' was not found.")
+
+    return directory_out_path
+
+
+def _validate_identifier(name: str, name_type: str) -> None:
+    if not is_python_identifier(name):
+        raise click.ClickException(
+            f"The {name_type} name '{name}' is not a valid Python identifier."
+        )
+
+
+def _get_interactive_module_and_class_names() -> Tuple[str, str]:
+    module_name = click.prompt("Enter a name for the python client module", type=str)
+    _validate_identifier(module_name, "module")
+    class_name = click.prompt("Enter a name for the python client class", type=str)
+    _validate_identifier(class_name, "class")
```

Comment:
In batch mode, these are optional, but in interactive mode they are required. That means using interactive mode requires the user to do more work than batch mode.

A good way to handle this is to calculate the default value based on the service class and include it in the prompt. If the user presses the enter key without entering a value, then use the default. `click.prompt` does this automatically if you specify `default=`.

Reply:
Create a default value and specified the same in prompt message for clarity.